docs/configurations/_index.md

```diff
@@ -1,13 +1,29 @@
 ---
 title: Configure Kafka cluster
 shorttitle: Configure
+aliases:
+  - /sdm/koperator/configurations/kafkacluster
```

Comment:
Here the alias means that previously this page had a different name, and since there is a change in the name, if someone tries to access a link with old name it gets redirected to this page.